tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi 'get_var';
+
+sub run {
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    while (defined(my $maintrepo = shift @repos)) {
```

Comment:
why this safety is required? why not just iterate `@repos`, `sub map_incidents_to_repo` should be responsible to give us a good mapping without undef values IMO, maybe already does...and relay in settings that we can assume always exist. I would simplify the code, specially in the test layer things should as simple as possible.

Reply:
I did what we discussed in the other comment, so this is done as well